python/tvm/tir/schedule/schedule.py

```diff
@@ -1289,7 +1289,10 @@ def cache_inplace(data_io: T.Buffer[64, "int32"]) -> None:
 
     @type_checked
     def cache_index(
-        self, block: Union[BlockRV, str], buffer_index: Union[int, str, Buffer]
+        self,
+        block: Union[BlockRV, str],
+        storage_scope: str,
+        cse_thresh: int,
```

Comment:
Can we set a default value for `cse_thresh`? e.g. 0/-1 for not enabling cse?

Reply:
> Can we set a default value for `cse_thresh`? e.g. 0/-1 for not enabling cse?

OK, I'll set 0 as caching all the index computation for the target block, should be useful for simple cases and expensive in complex cases.

Reply:
It's done. @Hzfengsy